rel/sys.config

```diff
@@ -1,4 +1,10 @@
 [
+ {sync, [
+        {src_dirs, {replace, [
+				{"../../applications", []},
```

Comment:
It is not enabled by default until you manually start it with sync:start() or sync:go(). What you see here is a mere app config, that could or could not be used.